test/lint/print-gitian-script.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
```

Comment:
Is this really necessary? Couldn't use some YAML processor like `yq`?

Reply:
There is no straightforward [way](https://mikefarah.github.io/yq/) to install `yq` on Travis, unfortunately.

Reply:
> Couldn't use some YAML processor like `yq`?

Done.
TIL, thank you ;)